conan/cli/commands/remote.py

```diff
@@ -177,7 +177,7 @@ def remote_login(conan_api, parser, subparser, *args):
                                 'requested interactively (not exposed)')
 
     args = parser.parse_args(*args)
-    remotes = conan_api.remotes.list(pattern=args.remote)
+    remotes = conan_api.remotes.list(pattern=args.remote, only_enabled=False)
```

Comment:
Could you please add a test checking that we're listing both enabled and disabled remotes?

Reply:
Listing is a different fix, in another place, and needs another test.

Reply:
Sorry, I did not realize this command was the remote login. Good then!